dataflow/snippets/batch_write_storage.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# [START dataflow_batch_write_to_storage]
+import apache_beam as beam
+from apache_beam.io.textio import WriteToText
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+def write_to_cloud_storage():
```

Comment:
How does this function know the output location or bucket name? As per test-cases, some arguments are sent to `sys.argv`, but this function does not seem to parse any sys.args

Passing the bucket name as an argument to this `write_to_cloud_storage(bucket_name: str):`

Reply:
This is idiomatic for Beam in Python. For example, [here](https://github.com/GoogleCloudPlatform/dataflow-cookbook/blob/main/Python/gcs/read_avro.py). The `PipelineOptions` class automatically parses the arguments. I added a code comment to try and clarify.

Reply:
Thank you.